action.yaml

```diff
@@ -54,6 +54,46 @@ runs:
 
         fi
 
+    - name: Test Report
+      uses: dorny/test-reporter@v1
+      id : test-result
+      if: ${{ env.cmd_type == 'UNIT_TEST' &&  env.test_report_path !='' && env.reporter !='' }}                   
+      with:
+        name: Unit Test                          
+        path: ${{ env.test_report_path}}              # '**/target/**/*.xml'   
```

Comment:
These new options need to be explained in the README file

Reply:
Ok, will update the readme file with new parameters.

Reply:
Updated Readme with new parameters